test/e2e/play_kube_test.go

```diff
@@ -1701,6 +1855,45 @@ func getPodNameInDeployment(d *Deployment) Pod {
 	return p
 }
 
+type Job struct {
+	Name        string
+	Labels      map[string]string
+	Annotations map[string]string
+	PodTemplate *Pod
+}
+
+func getJob(options ...jobOption) *Job {
+	j := Job{
+		Name:        defaultJobName,
+		Labels:      make(map[string]string),
+		Annotations: make(map[string]string),
+		PodTemplate: getPod(),
+	}
+	for _, option := range options {
+		option(&j)
+	}
+
+	return &j
+}
+
+type jobOption func(*Job)
+
+func withJobAnnotation(k, v string) jobOption {
+	return func(job *Job) {
+		job.Annotations[k] = v
+	}
+}
```

Comment:
@umohnani8 maybe you're OOTO this week or otherwise occupied? Anyhow when you get back to this, the latest lint failure is because this function is unused.

Reply:
Fixed